js/core/utils/icon.js

```diff
@@ -1,10 +1,16 @@
-var $ = require("../../core/renderer");
+import $ from "../../core/renderer";
 
-var getImageSourceType = function(source) {
+const ICON_CLASS = "dx-icon";
+
+const getImageSourceType = (source) => {
     if(!source || typeof source !== "string") {
         return false;
     }
 
+    if(/<\s*svg[^>]*>(.*?)<\s*\/\s*svg>/.test(source)) {
```

Comment:
`.*` between </svg> tag can include other tags. Replace it with `[^<]*`
Spaces after the `<` and before the `\` are possible only when you are using parsed data. You can remove it

```suggestion
    if(/<svg[^>]*>[^<]*<\/svg>/.test(source)) {
```